isb_web/sqlmodel_database.py

```diff
@@ -278,7 +278,7 @@ def paged_things_with_ids(
     thing_select = _base_thing_select(authority, status, limit, offset, min_id)
 
     if min_time_created is not None:
-        thing_select = thing_select.filter(Thing.tcreated >= min_time_created)
+        thing_select = thing_select.filter(Thing.tcreated > min_time_created)
```

Comment:
FWIW, this line seems less controversial.  I think I included it because I wasn't sure how accurate dates would be in the database, but it seems to do what we want even if it is limited to `>`.